### PR TITLE
Added toggle for Cart collapsed view

### PR DESCRIPTION
**Pull request by devYaoYH**
- Added toggle to switch between:
  - Expanded Cart view (renders `ItemCard`): Able to +/- adjust item amounts
  - Collapsed Cart view (renders `ItemCardCompact`): Not able to adjust item amounts
- Removed `CartActions.ts`, functions moved into `src/utils.ts`
- Removed debugging UI elements
- Removed FAB icon for toggle (replaced with `<Switch />`)
- Single FAB icon for checkout (to be replaced with "checkout" button as per mockups)
Screenshot 2020-06-29 at 18 52 20
Screenshot 2020-06-29 at 18 53 21

### CODE REVIEW

**Comment by BhavyaLight** [comment]
could you add any images for the changes here

**Comment by devYaoYH** [comment]
added screenshots with changes to description

File: client/src/components/ItemCard/ItemCard.tsx [hunk 1]
```diff
@@ -10,6 +12,9 @@ function ItemCard({
   cartItem: CartItem;
   updateItemQuantity: (barcode: string, quantity: number) => void;
 }) {
+  const theme = useTheme();
+  const themeSpacing = theme.spacing(1);
```
Comment (cszqwe) [inline client/src/components/ItemCard/ItemCard.tsx:16]: What does theme.spacing(1) refer to?

Comment (devYaoYH) [reply client/src/components/ItemCard/ItemCard.tsx:16]: https://github.com/googleinterns/scan-and-go/blob/97b27fe6b1a6cc5a931f7e553380fe6910a2f8c3/client/src/components/StoreCard/StoreCard.tsx#L19-L22
Think I need to move this comment somewhere more visible...

**Pull request merged**